app/webpacker/packs/admin.js

```diff
@@ -4,3 +4,6 @@ import { definitionsFromContext } from "stimulus/webpack-helpers";
 const application = Application.start();
 const context = require.context("controllers", true, /.js$/);
 application.load(definitionsFromContext(context));
+
+// Import style for flatpickr
+require("flatpickr/dist/flatpickr.css");
```

Comment:
Why here? What happened to the JS? The commit message doesn't explain anything.

Reply:
You're right, that should not be here. I followed the tutorial but it should be in `app/webpacker/css/admin/all.scss` (relatively on top to avoid CSS errors due to order)